docs/sources/tempo/configuration/_index.md

```diff
@@ -707,6 +710,34 @@ query_frontend:
 
 ```
 
+### Limit query size to improve performance and stability
```

Comment:
Isn't this the same feature as `Cap the maximum query length`?

Reply:
They are similar, but two different configuration parameters. one controls the size of the spanset query and the other sets the maximum length of a query.

Reply:
Clarified on Slack, this is an H3, the sections below are H4.

Reply:
The cap max query length heading is now a H4 instead of an H3.